src/rolling/file_number.rs

```diff
@@ -70,55 +97,29 @@ impl FileNumber {
     }
 
     pub fn filename(&self) -> String {
-        format!("wal-{:020}", self.inner.file_number)
+        format!("wal-{:020}", self.file_number)
```

Comment:
interesting... the format (20-digits) suggest that I was using 64 bits before (!?)

Reply:
yeah, I assumed the format using 20 digits meant u32 was a mistake, so I replaced every such u32 with u64